src/Microsoft.DotNet.Interactive.HttpRequest.Tests/Utility/AssertionExtensions.cs

```diff
@@ -22,4 +22,17 @@ internal static class AssertionExtensions
 
         return new AndWhichConstraint<ObjectAssertions, T>(subject.Should(), subject);
     }
+    
+    public static AndWhichConstraint<ObjectAssertions, T> ContainSingle<T>(
+        this GenericCollectionAssertions<HttpSyntaxNodeOrToken> should)
+        where T : HttpSyntaxNode
```

Comment:
Can we remove the overload on line 13 since `HttpSyntaxNodeOrToken` is a base type of `HttpSyntaxNode` (and since `IReadOnlyList` supports covariance)?